assets/base.css

```diff
@@ -2469,10 +2495,22 @@ menu-drawer + .header__search {
 }
 
 .header > .header__search {
-  grid-area: left-icon;
+  grid-area: left-icons;
   justify-self: start;
+  margin-left: 3.2rem;
 }
 
+.header:not(.drawer-menu) > .header__search {
+  margin-left: 0;
+}
+
+header-drawer {
+  grid-area: left-icons;
+  justify-self: start;
+}
```

Comment:
I still need this declaration

```
header-drawer {
  grid-area: left-icons;
}
```

Otherwise it looks like this
<img width="1211" alt="Screenshot 2023-02-24 at 12 46 18 PM" src="https://user-images.githubusercontent.com/105315663/221288276-6e1ec514-52eb-4a5b-b506-ddbf8c09121d.png">

Reply:
Let's replace it with 

```
.header--top-center header-drawer {
  grid-area: left-icons;
}
```